recipes/compas_libigl/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "compas_libigl" %}
+{% set version = "0.2.0" %}
+{% set hash = "4c8c61a681c046687ac13f3c3ac00c983b67d51d1f6d164c672b9563753f8c75" %}
+{% set pybind11_version = "2.5.0" %}
+{% set pybind11_hash = "97504db65640570f32d3fdf701c25a340c8643037c3b69aec469c10c93dc8504" %}
+{% set libigl_version = "2.2.0" %}
+{% set libigl_hash = "b336e548d718536956e2d6958a0624bc76d50be99039454072ecdc5cf1b2ede5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/compas-dev/compas_libigl/archive/v{{ version }}.tar.gz
+    fn: compas_libigl-{{ version }}.tar.gz
+    sha256: {{ libigl_hash }}
+  - url: https://github.com/pybind/pybind11/archive/v{{ pybind11_version }}.tar.gz
+    fn: pybind11-{{ pybind11_version }}.tar.gz
+    folder: ext/pybind11
+    sha256: {{ pybind11_hash }}
+  - url: https://github.com/libigl/libigl/archive/v{{ libigl_version }}.tar.gz
+    fn: libigl-{{ libigl_version }}.tar.gz
+    folder: ext/libigl
+    sha256: {{ libigl_hash }}
+
+build:
+  number: 0
+  skip: true  # [py<36]
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+
+requirements:
+  build:
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - eigen <3.4
+  host:
+    - pip
+    - python
+    - pybind11
+  run:
+    - python
+    - eigen <3.4
+    - boost-cpp
+    - nptyping
+    - typing_extensions
```

Comment:
Should this also require `igl` ?

Reply:
we don't use "their" binding if that is what you are asking